src/core/Form/Toggle/Toggle.tsx

```diff
@@ -54,7 +54,11 @@ export class Toggle extends Component<ToggleProps> {
     const { toggleStatus } = this.state;
 
     return (
```

Comment:
Should this file also contain `Toggle.withInput` and actually implement `variant = 'default' | 'withInput'` to `src/components`? (not boolean)

Reply:
Now in the recent commit, using the variant instead of the 'withInput' boolean.